extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/exec/SegmentLoadStatusFetcher.java

```diff
@@ -159,29 +159,18 @@ public void waitForSegmentsToLoad()
               return;
             }
 
-            Iterator<String> iterator = versionsToAwait.iterator();
-            log.info(
+            log.debug(
```

Comment:
Can we log.info this every minute ?

Reply:
And update the log message to say that even if the task is cancelled, the segments will continue to load.